frontend/src/components/Workspace/Experiment/Button/DeleteButton.tsx

```diff
@@ -41,12 +41,17 @@ export const DeleteButton = memo(function DeleteButton() {
     dispatch(deleteExperimentByUid(uid))
     uid === currentPipelineUid && dispatch(clearCurrentPipeline())
   }
+  const handleClosePopup = (event: KeyboardEvent) => {
+    if (event.key === "Escape") {
+      onClickCancel()
+    }
+  }
   return (
     <>
       <IconButton onClick={onClickOpen} disabled={isRunning} color="error">
         <DeleteOutlineIcon />
       </IconButton>
-      <Dialog open={open}>
+      <Dialog open={open} onClose={onClickCancel} onKeyDown={handleClosePopup}>
```

Comment:
MUI's Dialog can be closed by esc by default.

`frontend/src/Theme.ts`
has 
```
  MuiDialog: {
    defaultProps: {
      disableEscapeKeyDown: true,
    },
  },
```

So removing this would be enough, I think.

Reply:
@ReiHashimoto Yes, I removed the onKeyDown event and replaced disableEscapeKeyDown: false, please check it again.